<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SellerInfoForm</class>
 <widget class="QWidget" name="SellerInfoForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_headPic">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>128</width>
     <height>128</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>130</y>
     <width>270</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_name">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>名字:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_name">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pb_create">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>440</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	font-size:20px;
	background-image: url(:/images/loginpushButton.png);
 	background-position: center ;

}</string>
   </property>
   <property name="text">
    <string>创建书店</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_logout">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	font-size:20px;
	background-image: url(:/images/loginpushButton.png);
 	background-position: center ;

}</string>
   </property>
   <property name="text">
    <string>注销</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_info">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>180</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	font-size:20px;
	background-image: url(:/images/loginpushButton.png);
 	background-position: center ;

}</string>
   </property>
   <property name="text">
    <string>修改信息</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_pawd">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>270</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	font-size:20px;
	background-image: url(:/images/loginpushButton.png);
 	background-position: center ;

}</string>
   </property>
   <property name="text">
    <string>修改密码</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_pawdOld">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>447</x>
     <y>269</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_pawdNew">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>446</x>
     <y>318</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_pawdNew">
   <property name="geometry">
    <rect>
     <x>319</x>
     <y>309</y>
     <width>120</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>新密码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_pawdOld">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>260</y>
     <width>120</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>旧密码：</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>360</y>
     <width>1021</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_id">
   <property name="geometry">
    <rect>
     <x>328</x>
     <y>82</y>
     <width>71</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ID   :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_id">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>406</x>
     <y>91</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_sex">
   <property name="geometry">
    <rect>
     <x>331</x>
     <y>181</y>
     <width>90</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>性别：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_sex">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>408</x>
     <y>190</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>438</y>
     <width>332</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_storeId">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>STOREID  :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_id_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>490</y>
     <width>265</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_storeName">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>店名  :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_storeName">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
